.github/workflows/e2e-test-darts-cifar10.yaml

```
@@ -33,6 +33,6 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        kubernetes-version: ["v1.28.7", "v1.29.2", "v1.30.7"]
+        kubernetes-version: ["v1.28.7", "v1.29.2", "v1.30.7", "v1.31.3"]
```

```suggestion
        kubernetes-version: ["v1.29.2", "v1.30.7", "v1.31.3"]
```

I don't think that we want to support 1.28 in Katib. In the training-operator, the previous version is the last version for the v1 API. So, we added 1.28 to the support matrix.

---

SGTM.